services/dask-sidecar/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# DEVELOPMENT recipes for Workflows manager service
+#
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+
+TEMP_DIR := $(shell mktemp -d -t dask-docker-XXX)
+
+PHONY: build-dask
+#
+# NOTE:  At this moment, this does not seem to work https://docs.docker.com/engine/reference/commandline/build/
+# export DOCKER_BUILDKIT=0; docker build --tag local/dask:master https://github.com/dask/dask-docker.git#:base
```

Comment:
you should use buildx instead... it is now almost the default. did you try? ```docker build buildx```
you can also use buildx by default by doing something similar to docker buildx install (to be verified)

Reply:
NOTE that this recipe is to build the "official" dask from the repo! 
It does not work with buildx either
the problem is that, as I understand from the [doc](https://docs.docker.com/engine/reference/commandline/build/), we could build from a ``Dockerfile`` in a repo's folder as

```command line
export DOCKER_BUILDKIT=0
docker build --tag local/dask:master https://github.com/dask/dask-docker.git#:base
```
for [this dockerfile](https://github.com/dask/dask-docker/tree/main/base)

but it does not work ... :-(